core/buildTokens.js

```diff
@@ -0,0 +1,147 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const StyleDictionary = require('style-dictionary');
```

Comment:
Does `style-dictionary` have any provided types that we could use JSDoc annotations to import the type references for?

Reply:
Hum, don't know, will try to check that!

Reply:
The have ts types: https://github.com/amzn/style-dictionary/tree/main/types, would that be useful for JSDoc?